README.md

```diff
@@ -1 +1,12 @@
-# angular-using-scope-apply-lab
\ No newline at end of file
+# Using $scope.$apply to force $digest cycles
+
+## Objectives
+
+- Describe why our native APIs don't update the View
+- Use $apply to force a $digest to run a Model and View sync
+
+## Instructions
+
+The directive in this repo is broken - it's never updated. Let's change that.
+
+In our link function, add an event listener to listen to whenever the user presses a key on the page. Then, when they have, update the counter to reflect how many keys they have pressed.
```

Comment:
Do you mean like clicking a specific button on the page? Or would this count keystrokes? If it's the first, let's include the button in the directive.